Jenkinsfile

```diff
@@ -0,0 +1,6 @@
+@Library('linkurious-shared')_
```

Comment:
what is the trailing underscore for?

Reply:
Not a typo :) https://www.jenkins.io/doc/book/pipeline/shared-libraries/

Look for the section on Annotation (no anchor)